_data/domains.yml

```diff
@@ -213,7 +213,9 @@ websites:
       - sms
       - software
       - hardware
```

Comment:
Looks like they don't support any hardware methods other than U2F.
```suggestion
```

Reply:
So we only add specific tags when a site supports more than one method?

Reply:
the hardware tag is a catch all for every single hardware method. Since you added the U2F tag already, and they don't provide anything other than U2F, then the hardware tag is redundant.
Does that make sense?

Reply:
@Carlgo11 Got it. I don't think I'm a believer in the approach (confusing mobile UX) but I'll defer to those who've been working on this much longer than me!